src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+                        String task = arrOfStr[0];
+                        String deadline = arrOfStr[1];
+                        Task added = new Deadline(task,deadline);
+                        arr.add(added);
+                        System.out.println(lines + "\n" + "Ok I've added: " + added +
+                                "\nThere are " + arr.size() + " task(s) in the list now." + "\n" + lines);
+                        //echo = sc.nextLine();
+
+                    } else if (echo.startsWith("event")) {
+                        String str = echo.substring(6);
+                        String[] arrOfStr = str.split("/at");
+                        String task = arrOfStr[0];
+                        String deadline = arrOfStr[1];
+                        Task event = new Event(task,deadline);
+                        arr.add(event);
+                        System.out.println(lines + "\n" + "Ok I've added: " + event +
+                                "\nThere are " + arr.size() + " task(s) in the list now." + "\n" + lines);
+                        //echo = sc.nextLine();
+
+                    } else if (echo.startsWith("todo")) {
+                        try {
+                            String str = echo.substring(5);
+                            Task todo = new Todo(str);
+                            arr.add(todo);
+                            System.out.println(lines + "\n" + "Ok I've added: " + todo +
+                                    "\nThere are " + arr.size() + " task(s) in the list now." + "\n" + lines);
+                            //echo = sc.nextLine();
+                        } catch (StringIndexOutOfBoundsException e) {
+                            System.out.println(lines + "\n" + "☹ OOPS!!! The description of a todo cannot be empty."
+                                + "\n" + lines);
+                            //echo = sc.nextLine();
+                        }
+
+                    } else if (echo.startsWith("delete")) {
+                        int number = Integer.parseInt(echo.substring(7));
+                        System.out.println("im here");
+                        Task task = arr.get(number - 1);
+                        arr.remove(number - 1);
+                        System.out.println(lines + "\n" + "Ok I've deleted this task: " + task +
+                                "\nThere are " + arr.size() + " task(s) in the list now." + "\n" + lines);
+                    }
+                    else {
+                        try {
+                            throw new IllegalArgumentException();
+                        } catch (IllegalArgumentException e){
+                            System.out.println(lines + "\n" + "☹ OOPS!!! I'm sorry " +
+                                    "but I don't know what that means." + "\n" + lines);
+                        }
+                      }
+                    }
+                        echo = sc.nextLine();
```

Comment:
```suggestion
                    }
                    echo = sc.nextLine();
```
Indentation of echo should be the same as the last braces